packages/components/src/Card/cardelevations.css

```diff
@@ -0,0 +1,17 @@
+.lowElevation,
```

Comment:
I know that in this folder, `cardcolors.css` exists, but we should be using pascal case for our file names.  Don't worry about updating `cardcolors` but if we can update these new files to `CardElevations.css` along with the d.ts file, that would be great!

Also, I think Jacob just merged a PR which makes each css file a module file, and that's leading to some merge conflicts. Might want to pull that in and update this at the same time.

Reply:
Ah great call, I was definitely basing my naming off of `cardcolors.css` but it makes sense that I should be using pascal case based on what I have seen elsewhere in the codebase. I was waiting until I saw if there were other changes required before doing the rebase but I will do both now